tests/integration/kubernetes/confidential_kbs.sh

```diff
@@ -308,16 +321,11 @@ function kbs_k8s_deploy() {
 
 	echo "::group::Deploy the KBS"
 	if [ "${KATA_HYPERVISOR}" = "qemu-tdx" ]; then
-		echo "Setting up custom PCCS for TDX"
-		cat <<- EOF > "${COCO_KBS_DIR}/config/kubernetes/custom_pccs/sgx_default_qcnl.conf"
-{
- "pccs_url": "https://$(hostname -i | grep -o "^[0-9.]*"):8081/sgx/certification/v4/",
-
- // To accept insecure HTTPS certificate, set this option to false
- "use_secure_cert": false
-}
-EOF
-		export DEPLOYMENT_DIR=custom_pccs
+		echo "::group::Setting up ITA/ITTS for TDX"
+		pushd "${COCO_KBS_DIR}/config/kubernetes/ita/"
+			sed -i -e "s/tBfd5kKX2x9ahbodKV1.../${ITA_KEY}/g" kbs-config.toml
```

Comment:
The string `tBfd5kKX2x9ahbodKV1` is always constant?
Could you add a comment mentioning which `property` of the KBS config file is being affected by this line?
[KBS Configuration File]: https://github.com/confidential-containers/trustee/blob/main/kbs/docs/config.md

Reply:
The string is always constant.

However, the documentation you pointed out doesn't apply for ITA / ITTS.  The `sed` replaces https://github.com/confidential-containers/trustee/blob/3b7e73d760feb8650db7c88b19c5e1bf9968d071/kbs/config/kubernetes/ita/kbs-config.toml#L13

I'd prefer adding this as a follow-up (instead of adding it here and then having to wait a few more hours for the CI to run again), but it can only be added after we expand the KBS documentation for the kustomizations.

Can you open an issue on Trustee to add docs for the kustomizations?  Then we do it there, and reflect the changes here as well.

Reply:
Actually there's doc for ITA: https://github.com/confidential-containers/trustee/blob/main/kbs/docs/config.md#intel-trust-authority-formerly-known-as-amber

I'd still prefer adding this as a follow-up, just for the sake of not waiting for 1~2 hours before having this one merged.

@GabyCT, can you work on the follow-up once this one is merged?

Reply:
Argh, as the metrics CI is failing, let me do this change now.